src/components/TwitchSidebar/TwitchSidebar.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+
+import {
+  EuiPanel,
+  EuiImage,
+  EuiText,
+  EuiListGroup,
+  EuiListGroupItem,
+} from "@elastic/eui";
+import {  liveStreams } from './Streams'
+
+/* Styles */
+import "./_twitch-sidebar.scss";
+
+/* Types */
+import { TwitchStream } from "../../models/TwitchStream";
+
+export const TwitchSidebar: FunctionComponent = () => {
+  const [streams, setStreams] = useState<TwitchStream[]>([])
+  useEffect(() => {
```

Comment:
then you can do 
```
const {...rest} = props
```

Reply:
Not sure the point of this step? I can reference off of props instead of spreading it all into an object and then referencing that object